tests/managed-service/test_sanity_ms.py

```diff
@@ -0,0 +1,124 @@
+import logging
+from time import sleep
+import pytest
+
+from ocs_ci.helpers.sanity_helpers import SanityManagedService
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    libtest,
+    ManageTest,
+    ignore_leftovers,
+    managed_service_required,
+)
+
+log = logging.getLogger(__name__)
+
+
+@libtest
+@ignore_leftovers
```

Comment:
Why are leftovers ignored? What is created and not deleted as part of those tests?

Reply:
Yes, you're right. I removed it.